.gitignore

```diff
@@ -11,3 +11,4 @@ Gemfile.lock
 # rspec failure tracking
 .rspec_status
 /.envrc
+*.gem
```

Comment:
Aren't the built gems going into `/pkg`, which is covered by the entry on line 6?

Reply:
I removed this, using `rake gem` instead means the file goes into the /pkg folder.